Spigot-Server-Patches/0582-Fix-Wither-Not-Hostile-To-Plaers-Bug.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheShermanTanker <tanksherman27@gmail.com>
+Date: Sun, 20 Sep 2020 22:25:26 +0800
+Subject: [PATCH] Fix-Wither-Not-Hostile-To-Plaers-Bug
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityWither.java b/src/main/java/net/minecraft/server/EntityWither.java
+index 29bb74e51e73ab5b1ccc814b6599e0baf4d8094c..0e31c32e48382bbd603f0ac4b3ebd751c9577821 100644
+--- a/src/main/java/net/minecraft/server/EntityWither.java
++++ b/src/main/java/net/minecraft/server/EntityWither.java
+@@ -41,6 +41,10 @@ public class EntityWither extends EntityMonster implements IRangedEntity {
+         this.f = 50;
+     }
+ 
++    private static Predicate<EntityLiving> getWitherTargetEvaluatorNotUndead() { //Paper - OBFHELPER
++    	return by;
++    }
++
+     @Override
+     protected void initPathfinder() {
+         this.goalSelector.a(0, new EntityWither.a());
+@@ -49,7 +53,8 @@ public class EntityWither extends EntityMonster implements IRangedEntity {
+         this.goalSelector.a(6, new PathfinderGoalLookAtPlayer(this, EntityHuman.class, 8.0F));
+         this.goalSelector.a(7, new PathfinderGoalRandomLookaround(this));
+         this.targetSelector.a(1, new PathfinderGoalHurtByTarget(this, new Class[0]));
+-        this.targetSelector.a(2, new PathfinderGoalNearestAttackableTarget<>(this, EntityInsentient.class, 0, false, false, EntityWither.by));
++        this.targetSelector.a(2, new PathfinderGoalNearestAttackableTarget<>(this, EntityInsentient.class, 0, false, false, getWitherTargetEvaluatorNotUndead()));
++        this.targetSelector.a(2, new PathfinderGoalNearestAttackableTarget<>(this, EntityHuman.class, 0, false, false, getWitherTargetEvaluatorNotUndead())); //Fix Wither not hostile to Players
```

Comment:
tag with // Paper, as well as remove the predicate, players can't be undead.

I also question, should this be ABOVE Insentient so it prefers players first?
Also worth considering a config option for those "MAH VANILLA" people.